package.json

```diff
@@ -40,7 +40,10 @@
     "autoprefixer": "^10.4.16",
     "eslint-config-prettier": "^9.0.0",
     "fast-glob": "^3.3.1",
+    "install": "^0.13.0",
+    "npm": "^10.2.0",
```

Comment:
Why do we need these packages? It looks like you accidentally installed them

Reply:
i am sorry , i will remove it